### DISCUSSION TITLE
Step stuck

### DISCUSSION THREAD

**Discussion by Raksa197** [skills/.github#298] [Code with Copilot]
Hi. In step 1, I can finish the steps before Step 6 Click the Code button located in the middle of the page. At step 6 I stuck because I cannot find where is the code button in the middle of the page. 

<img width="1440" alt="Screenshot 2023-07-20 at 2 08 25 PM" src="https://github.com/skills/.github/assets/97582234/517a2552-c433-46f2-a6ce-fe3d2f657a27">

**Comment by sinsukehlab** [comment]
Click the green button **`<> Code ▾`**.

<img width="1440" alt="254802380-517a2552-c433-46f2-a6ce-fe3d2f657a27" src="https://github.com/skills/.github/assets/94058150/6563686f-5ac5-453f-a9bf-840988db8c1e">

**Reply by AbulKhayer2244** [reply]
Hi
